### DISCUSSION TITLE
Roadblock right clic options

### DISCUSSION THREAD

**Discussion by mathmaj** [Keriew/augustus#306] [Ideas]
Hi

When we right click on a roadblock, a first window is opening, where very few informations are writen. If you want to block acces or accepting access to a specific character, you have to access to a second window.
Don't you think we could acces to the second window directly?

Still regarding the roadblock: By default, in the roadblock options window, every type of character are blocked (wich is logical). But if I want to accept everyone except a single type of character, I have to check all the boxes one by one. What about a check box to accept everyone? Then we can only refuse one. It would saves time!

What do you think?

**Comment by crudelios** [comment]
The first issue is already consistant with the rest of the UI, but we can think about it. Maybe we could revamp the entire thing eventually.

Regarding the second suggestion, seems like a good idea!
